app/controllers/user_profiles_controller.rb

```diff
@@ -59,6 +60,12 @@ def delete_profile_image
     redirect_to controller: 'user_profiles', action: 'show', username: @user.username
   end
 
+  def max_project
```

Comment:
This is interesting! I hadn't thought about doing anything with the user profile pages. I would prefer this feature — switching terminology on user profile pages — as a separate PR.

Reply:
Will do. 
And I have a question here. In case that there are more wikis with max occurrences, Wikipedia will be prioritized and articles version rendered.
If there is no Wikipedia, the first in the max array will render.
What do you think to check for Wikidata after Wikipedia (make Wikidata second priority) and render Items version if, for example, max array has Wikidata and other Wikis, but no Wikipedia?

Reply:
That sounds reasonable, although perhaps not a high enough priority to worry about. I think this `max projects` approach should only be a temporary one. What this user profile page really wants is to show Wikidata stats separately from Wikipedia stats, rather than showing combined stats from multiple projects with just the labels changing based on which project is the most used.

Reply:
if that's the case, no need for that